src/main/java/com/monumental/util/csvparsing/MonumentBulkValidationResult.java

```diff
@@ -0,0 +1,49 @@
+package com.monumental.util.csvparsing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class for holding information about a Bulk Monument Validation operation
+ */
+public class MonumentBulkValidationResult {
+
+    private Map<Integer, CsvMonumentConverterResult> results = new HashMap<>();
+
+    private String error;
+
+    public Map<Integer, CsvMonumentConverterResult> getResults() {
+        return this.results;
+    }
+
+    public void setResults(Map<Integer, CsvMonumentConverterResult> results) {
+        this.results = results;
+    }
+
+    public String getError() {
+        return this.error;
+    }
```

Comment:
This getter is not used. I'm guessing this is for the "general" errors you talked about? Do we still want this?

Reply:
It'll be used in v2